pytorch_lightning/loggers/tensorboard.py

```diff
@@ -109,8 +109,18 @@ def log_hyperparams(self, params: Union[Dict[str, Any], Namespace]) -> None:
                 " hyperparameter logging."
             )
         else:
+            # See https://github.com/PyTorchLightning/pytorch-lightning/ \
+            # pull/609#issuecomment-598253152
+            # Passing only those params which tensorboard allows here:
+            # https://github.com/pytorch/pytorch/blob/master/torch/utils/tensorboard/summary.py#L134
+            from six import string_types
             from torch.utils.tensorboard.summary import hparams
-            exp, ssi, sei = hparams(params, {})
+            tensorboard_params = {}
+            for k, v in params.items():
+                if isinstance(v, int) or isinstance(v, float) or isinstance(v, string_types) \
```

Comment:
This can (and should) be simplified to `isinstance(v, (int, float, ...))` for better readability.